holopy/core/utils.py

```diff
@@ -161,6 +162,25 @@ def repeat_sing_dims(indict, keys = 'all'):
 
     return updated(indict, subdict)
 
+class LnpostWrapper(HoloPyObject):
```

Comment:
Is this tested in holopy right now?

I realize that you're just moving code from one file to another, so writing unit tests for this is above and beyond the call of duty. Do we want to make a list of objects though that aren't tested, or no?

Reply:
Ditto what @briandleahy said

Reply:
I confirmed there are no tests for this object, but I'm not writing them for this PR.